funnel/utils/markdown/base.py

```diff
@@ -0,0 +1,137 @@
+"""Base files for markdown parser."""
+
+from copy import deepcopy
+from typing import Any, List, Mapping, Optional, Type, Union, overload
+
+from markdown_it import MarkdownIt
+from markupsafe import Markup
+from mdit_py_plugins import anchors, footnote, tasklists
+from typing_extensions import Protocol
+
+from coaster.utils import make_name
+from coaster.utils.text import VALID_TAGS, normalize_spaces_multiline, sanitize_html
+
+MARKDOWN_HTML_TAGS = deepcopy(VALID_TAGS)
+
+
+# --- Typing Protocol for markdown-it-py plugins ---------------------------------------
+class MarkdownItPluginProtocol(Protocol):  # pylint: disable=too-few-public-methods
```

Comment:
This disable should not be necessary as it's already in the global list in `pyproject.toml`.